environment.yml

```diff
@@ -28,7 +29,7 @@ dependencies:
   - sphinx-automodapi
   - sphinx-copybutton
   - pip:
-    - kaleido
-    - plotly
-    - bump2version
-    - -e .
+      - kaleido
```

Comment:
We should be consistent with indentation of yaml-files everywhere. Is that what this change is about?

Reply:
Good point to keep it consistent. I reverted the change and adjusted my code formatter.